tests/helpers/setup-fetch-record.js

```diff
@@ -29,9 +29,11 @@ setupFetchRecord.beforeEach = function(options = {}) {
 
 setupFetchRecord.afterEach = function() {
   afterEach(async function() {
-    this.polly.pause();
-    await this.fetchRecord({ method: 'DELETE' });
-    this.polly.play();
+    if (this.polly) {
```

Comment:
Possible for test suite to fail before `this.polly` is set and made the stderr annoying to scroll through since we always assume it was.

Reply:
Might be good to add an inline comment about this.

Reply:
Done